get.sh

```diff
@@ -149,12 +149,15 @@ getBinaryOpenjdk()
 	fi
 	
 	if  [ "${download_url}" != "" ]; then
-		echo "curl -OLJks ${curl_options} "${download_url}""
-		curl -OLJks ${curl_options} "${download_url}"
-		if [ $? -ne 0 ]; then
-			echo "Failed to retrieve the jdk binary, exiting"
-			exit 1
-		fi
+		for file in $download_url
+		do 
+			echo "curl -OLJks ${curl_options} $file"
+			curl -OLJks ${curl_options} $file
+			if [ $? -ne 0 ]; then
+				echo "Failed to retrieve the jdk binary, exiting"
```

Comment:
Should update both line 44 (with a note about format of customizedURL, if multiple files present or a note about the fact its supported) and line 157, to be more useful, failed to retrieve $file, or something like that.

Reply:
Updates made.